apps/server/lib/lexical/server/code_intelligence/completion/translations/callable.ex

```diff
@@ -71,7 +71,13 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.Callable do
 
   defp sort_text(%_{name: name, arity: arity}) do
     normalized = String.replace(name, "__", "")
-    "#{normalized}/#{arity}"
+    fun = "#{normalized}/#{arity}"
+
+    if String.starts_with?(name, "__") or name in ["module_info"] do
+      fun
+    else
+      Env.boost(fun)
+    end
```

Comment:
The logic in this section seems convoluted, mainly because the `normalization` and `conditional statements` are far apart and not connected.

Therefore, I prefer some code like this:

```elixir
  defp sort_text(%_{name: name, arity: arity}) do
    if String.starts_with?(name, "__") or name in ["module_info"] do
      "#{normalize(name)}/arity"
    else
      boosted = name |> normalized() |> Enum.boost
      "#{boosted}/arity"
    end
  end

  defp normalize(name) do
    String.replace(name, "__", "")
  end
```

Reply:
I think what was there is fine. If you follow the suggestion, you have two places where normalizations happen and the logic to build the sort text is duplicated

Reply:
Ok, ignore my suggestion.